protocol/directory.go

```diff
@@ -31,15 +36,18 @@ func InitDirectory(epDeadline merkletree.TimeStamp, vrfKey vrf.PrivateKey,
 	return d
 }
 
-func (d *ConiksDirectory) Update() {
+func (d *ConiksDirectory) Update(up *UpdatePolicies) {
 	d.pad.Update(d.policies)
 	// clear issued temporary bindings
 	for key := range d.tbs {
 		delete(d.tbs, key)
 	}
+	if up != nil {
+		d.setPolicies(up.EpDeadline, up.VrfKey)
```

Comment:
Can you elaborate on that? If we change the `vrfKey` at epoch `t=2`,  then the `pad` is updated at epoch `t=3` with the old `vrfKey` and next, we update the directory's policies with new key. Thus, the new `vrfKey` will be used at epoch `t=4`, right?